app/Notifications/SendCheckmateStats.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Project;
+use Illuminate\Notifications\Notification;
+use NathanHeffley\LaravelSlackBlocks\Messages\SlackMessage;
+
+class SendCheckmateStats extends Notification
+{
+    public function via($notifiable)
+    {
+        return ['slack'];
+    }
+
+    public function toSlack($notifiable)
+    {
+        $message = (new SlackMessage)
+            // Provide a text-only fallback message
+            ->content(sprintf('Here are your Checkmate stats! %s', config('app.url')))
+            ->block(function ($block) {
+                $block
+                    ->type('section')
+                    ->text([
+                        'type' => 'mrkdwn',
+                        'text' => 'Here are your Checkmate stats!',
+                    ])
+                    ->accessory([
+                        'type' => 'button',
+                        'url' => config('app.url'),
+                        'text' => [
+                            'type' => 'plain_text',
+                            'text' => 'Open Checkmate',
+                            'emoji' => true,
+                        ],
+                    ]);
+            });
+
+        app(Project::class)->all()
+            ->filter(function($project) {
+                return ! config('app.show_private_repos')
+                    ? $project->status !== Project::STATUS_CURRENT && ! $project->is_private
+                    : $project->status !== Project::STATUS_CURRENT;
+            })
+            ->sortByDesc('status')
+            ->each(function ($project) use ($message) {
```

Comment:
Maybe extract this out to a `appendProjectToMessage` method or something similar?

so it would be like:

```php
    ->each(function ($project) use ($message) {
        $this->appendProjectToMessage($message, $project);
    });
```

And then refactor out all the content to a protected/private method